src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -1043,6 +1043,19 @@ internal bool GetPhysicalFileFromServer(string serverPath, string pdbIndexPath,
                             // So set it (this is what the symsrv code on Windows sets).   On NetStandard1.6 we give up since we dont' have this API available.  
                             req.UserAgent = "Microsoft-Symbol-Server/6.13.0009.1140";
 #endif
+
+#if NET45
+                            // Ugly support for ADO sources
+                            if (fullUri.StartsWith("https://") && fullUri.Contains("artifacts.visualstudio.com/"))
```

Comment:
Can you share what the URL structure looks like?  If I recall correctly its something like "https://<something>.artifacts.visualstudio.com/..."  If so, I'm thinking it would be good to change this to `fullUri.Host.EndsWith("artifacts.visualstudio.com", StringComparison.OrdinalIgnoreCase)`

Reply:
Yes, but we still need to check the scheme (so as to never send a security token to an unverified target or through an unencrypted channel)

Reply:
Agreed.

Reply:
There is also a second url that is equivalent
https://<orgName>.artifacts.visualstudio.com/defaultcollection/_apis/symbol/symsrv/
https://artifacts.dev.azure.com/<orgName>/_apis/symbol/symsrv/

I remember noting it down from the DevOps documentation. Both work for me.